src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.exception.ArgumentMustBeNumException;
+import duke.io.FileIO;
+import duke.io.Printer;
+import duke.task.Task;
+import duke.task.TaskList;
+
+public class DeleteCommand extends Command {
+  private String s;
+
+  public DeleteCommand(Printer out, TaskList taskList, FileIO savefile, String name) {
+    super(out, taskList, savefile);
+    this.s = name;
+  }
+
+  @Override
+  public void action() {
+    Task task;
+    try {
+      int i = Integer.parseInt(s); // 1-indexed
```

Comment:
Not sure if this is OK under the Java coding standard, but perhaps can indent the comment under this line.